clmm/theory/numcosmo.py

```diff
@@ -116,6 +116,11 @@ def _get_einasto_alpha(self, z_cl=None):
         # Note that z_cl is needed for CCL<2.6 only
         return self.hdpm.props.alpha
 
+    def _get_delta_mdef_virial(self, z_cl):
+        "Gets the overdensity"
+        # This fuction can return the overdensity for all mass definitions
```

Comment:
The comment description does not match that of ccl.py and the docstring in parent_class.py. Is it because of a more general behaviour of the `Delta` function in numcosmo? 

In any case, I believe that this function will be called only when mdef=virial? In that case, I'd suggest expanding the comment to explain that the NC function can do more, but that we restrict the usage to mdef=virial in CLMM. Otherwise the function name is at odds with the comment and I find this confusing.

Reply:
Fixed.